errorreporting/src/main/java/com/example/errorreporting/QuickStart.java

```diff
@@ -18,52 +18,69 @@
 
 // [START error_reporting_quickstart]
 // [START error_reporting_setup_java]
-
 import com.google.cloud.ServiceOptions;
-import com.google.devtools.clouderrorreporting.v1beta1.ErrorContext;
 import com.google.devtools.clouderrorreporting.v1beta1.ProjectName;
 import com.google.devtools.clouderrorreporting.v1beta1.ReportErrorsServiceClient;
 import com.google.devtools.clouderrorreporting.v1beta1.ReportedErrorEvent;
-import com.google.devtools.clouderrorreporting.v1beta1.SourceLocation;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.io.StringWriter;
 
 /**
- * Snippet demonstrates using the Error Reporting API to report a custom error event.
+ * Snippet demonstrates using the Error Reporting API to report a custom error
+ * event.
  *
- * <p>This library is not required on App Engine, errors written to stderr are automatically written
- * to Error Reporting. It is also not required if you are writing logs to Cloud Logging. Errors
- * written to Cloud Logging that contain an exception or stack trace are automatically written out
+ * <p>
+ * This library is not required on App Engine, errors written to stderr are
+ * automatically written
```

Comment:
Nit: we addressed the long lines, but I think we created extra short lines in the process. Is there a way to make them more balanced?